pages/demo/test.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="block h-screen w-full">
+    <input ref="place" v-model="value" class="h-5 w-1/2 bg-gray-200" />
+    <button class="bg-blue-500" @click="save">{{ title }}</button>
+    <div id="map" class="mt-5 h-1/2 w-1/2 bg-gray-500"></div>
```

Comment:
it shouldn't require map

Reply:
it means we have to not use `id` property in `div` tag?